python/tk_framework_desktopserver/shotgun/scripts/cache_commands.py

```diff
@@ -255,7 +255,7 @@ def cache(
     with open(arg_data_file, "rb") as fh:
         arg_data = cPickle.load(fh)
 
-    sys.path = arg_data["sys_path"]
+    sys.path.extend(arg_data["sys_path"])
```

Comment:
Good question. I've been going back-and-forth on that in my head since yesterday.

I think I've convinced myself that you're correct, and that it should be prepended. That will ensure that the sgtk imported here prior to bootstrap is the same tk-core as is running in Desktop. I think that's the most important thing.